lib/manager/components/ProjectSettingsForm.js

```diff
@@ -330,22 +333,18 @@ export default class ProjectSettingsForm extends Component<Props, State> {
                     type='text'
                     defaultValue={model.bounds
                       ? `${model.bounds.west},${model.bounds.south},${model.bounds.east},${model.bounds.north}`
-                      : ''
-                    }
+                      : ''}
                     ref='boundingBox'
-                    placeholder={this.messages('fields.location.boundingBoxPlaceHolder')}
-                    onChange={this._onChangeBounds} />
-                  {
-                    <InputGroup.Button>
-                      <Button
-                        // TODO: wait for react-leaflet-draw to update library
-                        // to re-enable bounds select
-                        disabled
-                        onClick={this._onOpenMapBoundsModal}>
-                        <Glyphicon glyph='fullscreen' />
-                      </Button>
-                    </InputGroup.Button>
-                  }
+                    placeholder={this.messages(
+                      'fields.location.boundingBoxPlaceHolder'
+                    )}
+                    onChange={this._onChangeBounds}
+                  />
+                  {<InputGroup.Button>
```

Comment:
Did you mean to delete the todo comment here? This button is permanently disabled it seems.

Reply:
Good eye! Looks like the auto-formatter swallowed it up. Added and extended the comments in f6d26732